src/commands/dev/lib/dev-builder.ts

```diff
@@ -167,21 +167,26 @@ export async function executeBuild(
     meta: { isDev: true, requestPath, filesChanged, filesRemoved }
   };
 
-  buildProcess.send({
-    type: 'build',
-    builderName: pkg.name,
-    buildParams
-  });
+  let buildResultOrOutputs: BuilderOutputs | BuildResult;
+  if (buildProcess) {
+    buildProcess.send({
+      type: 'build',
+      builderName: pkg.name,
+      buildParams
+    });
 
-  const buildResultOrOutputs = await new Promise((resolve, reject) => {
-    buildProcess!.once('message', ({ type, result }) => {
-      if (type === 'buildResult') {
-        resolve(result);
-      } else {
-        reject(new Error(`Got unexpected message type: ${type}`));
-      }
+    buildResultOrOutputs = await new Promise((resolve, reject) => {
+      buildProcess!.once('message', ({ type, result }) => {
```

Comment:
```suggestion
      buildProcess.once('message', ({ type, result }) => {
```

You don't need the bang anymore now that you have an `if`

Reply:
It doesn't work inside the Promise function for some reason.